docker/test/README.md

```diff
@@ -0,0 +1,34 @@
+Docker image for Huginn testing
+=================================================
+
+This image allows the [Huginn](https://github.com/cantino/huginn) test suite to be run in a container, against multiple databases.
+
+It was inspired by the [official docker container for huginn](https://registry.hub.docker.com/u/cantino/huginn)
+
+In Development Mode, the source code of the current project directory is mounted as a volume overlaying the packaged `/app` directory.
+
+## Development Usage
+
+Build a docker image that contains huginn, as well as huginn test dependencies using `docker-compose`:
+    cd docker/test
+    docker-compose -f develop.yml build
+
+Run all specs against a MySQL database using `docker-compose`:
+
+    cd docker/test
+    docker-compose -f develop.yml run huginn_test_mysql
+    docker-compose -f develop.yml down
+
+Run all specs against a Postgres database using `docker-compose`:
+
+    cd docker/test
+    docker-compose -f develop.yml run huginn_test_postgres
+    docker-compose -f develop.yml down
+
+Run a specific spec using `docker-compose`:
+    docker-compose -f develop.yml run huginn_test_postgres rspec ./spec/helpers/dot_helper_spec.rb:82
+    docker-compose -f develop.yml down
+
+## Usage
+
+*TODO*
```

Comment:
Do you plan do add more documentation? If not I think we could can the `usage` section.